src/admin-views/meta-box.php

```diff
@@ -128,8 +128,15 @@
 					<td width="20%"><label for="ticket_provider"><?php esc_html_e( 'Sell using:', 'event-tickets' ); ?></label></td>
 					<td>
 						<?php
-						$checked = true;
+						$checked        = true;
+						$default_module = null;
+
+						if ( method_exists( 'Tribe__Tickets__Tickets', 'get_default_module' ) ) {
```

Comment:
Is this `if` statement needed? We are currently in the event-tickets so this method should be known to exist.

Reply:
The method does not exist in the class currently; it's a TER method